### PR TITLE
DIGI_7_ListAllAvaliableMenuItems

### PR DESCRIPTION
**Pull request by praveen957**
1.added menu item model
2.added menu item Library
3. added menu item controller and view
4.modifications are done on css

### CODE REVIEW

File: DigiOrderEnterprise/DigiOrderEnterprise/Controllers/MenuItemController.cs [hunk 1]
```diff
@@ -1,17 +1,37 @@
...
+            try
+            {
+                menuItemLibrary = new MenuItemLibrary();
+                Task<HttpResponseMessage> response = menuItemLibrary.getAllMenuItems();
+                string data = response.Result.Content.ReadAsStringAsync().Result;
+                modelList = JsonConvert.DeserializeObject<List<MenuItemModel>>(data);
+            }
+            catch (Exception ex)
+            {
+                ViewBag.ErrorMessage = "web api is not responding";
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterprise/Controllers/MenuItemController.cs:30]: Change message to"Something went wrong.Contact administrator..'"

File: DigiOrderEnterprise/DigiOrderEnterprise/Controllers/MenuItemController.cs [hunk 2]
```diff
@@ -1,17 +1,37 @@
...
+            }
+            ViewBag.ControlNavigation = "MenuItem";
+            ViewBag.ControlAddNewNavigation = "~/MenuItem/Create";
+            return View(modelList);
         }
     }
-}
\ No newline at end of file
+}
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterprise/Controllers/MenuItemController.cs:37]: Add End of file

File: DigiOrderEnterprise/DigiOrderEnterprise/Views/MenuItem/Index.cshtml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+@{
+    ViewBag.Title = "Index";
+    Layout = "~/Views/Shared/_EnterpriseLayout.cshtml";
+}
+<div>
+    <div id="successmsg" style=" text-align: center;background-color:greenyellow; border:hidden;"> @TempData["Message"] </div>
+    <div id ="errormsg" style=" text-align: center;background-color:red; border:hidden">@ViewBag.ErrorMessage  </div>
+</div>
+<div class="entry value-plus active">&nbsp;</div>
+@{ if (Model != null)
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterprise/Views/MenuItem/Index.cshtml:11]: Check for condition model.count>0

File: DigiOrderEnterprise/DigiOrderEnterprise/Views/MenuItem/Index.cshtml [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+                </thead>
+                <tbody>
+                    @{
+                        int i = 1;
+                        foreach (var each in Model)
+                        {
+                            <tr class="rem1">
+                                <td id="serialnum" style="text-align:center">@i</td>
+                                <td id ="image"class="invert-image"><a href="single.html"><img src="~/asset/images/@each.IMAGE" style="width:50px; height:50px;" alt=" " class="img-responsive"></a></td>
+
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterprise/Views/MenuItem/Index.cshtml:35]: Remove space

File: DigiOrderEnterprise/DigiOrderEnterprise/Views/MenuItem/Index.cshtml [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+                                <tdid ="description">@each.DESCRIPTION</tdid>
+                                @if (each.TYPE == true)
+                                {
+                                    <td id="type" style="color:green">VEG </td>
+                                }
+                                else
+                                {
+                                    <td id ="type"style="color:red">NON-VEG</td>
+                                }
+
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterprise/Views/MenuItem/Index.cshtml:46]: removee spaces

File: DigiOrderEnterprise/DigiOrderEnterprise/Controllers/MenuItemController.cs (hunk 2 above)
Comment (rajendarreddy1234) [reply DigiOrderEnterprise/DigiOrderEnterprise/Controllers/MenuItemController.cs:37]: Resolved

File: DigiOrderEnterprise/DigiOrderEnterpriseLibrary/CategoryLibrary.cs [hunk 6]
```diff
@@ -17,7 +17,7 @@ public Task<HttpResponseMessage> getAllCategories(string EndPoint)
             try
             {
                 libraryUtility = new LibraryUtility();
-                httpResponse = libraryUtility.GetAPISync(EndPoint);
+                httpResponse = libraryUtility.ExecuteApiAsync(EndPoint);
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterpriseLibrary/CategoryLibrary.cs:20]: REVERT CATEGORY LIBRARY

File: DigiOrderEnterprise/DigiOrderEnterprise/Controllers/MenuItemController.cs (hunk 1 above)
Comment (rajendarreddy1234) [reply DigiOrderEnterprise/DigiOrderEnterprise/Controllers/MenuItemController.cs:30]: Resolved

**Pull request merged**